src/util/ssa_expr.cpp

```diff
@@ -55,13 +56,20 @@ static void build_ssa_identifier_rec(
   else if(expr.id()==ID_symbol)
   {
     os << to_symbol_expr(expr).get_identifier();
+    l1_object_os << to_symbol_expr(expr).get_identifier();    
```

Comment:
I would suggest to invoke `to_symbol_expr(expr).get_identifier()` only once and then copy the string into l1_object_os